test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_stabilityai.py

```diff
@@ -1,21 +1,150 @@
 from __future__ import absolute_import
 
+import os
 import sys
 
+from io import BytesIO
+from PIL import Image
 import pytest
+import sagemaker
+from sagemaker.pytorch import PyTorchModel
+from sagemaker.serializers import JSONSerializer
+from sagemaker.deserializers import BytesDeserializer
+
+import time
 import logging
 
+from ...integration import sdxl_gpu_path, sdxl_gpu_script
+
+
+from .timeout import timeout_and_delete_endpoint
+from .... import invoke_pytorch_helper_function
 
 LOGGER = logging.getLogger(__name__)
 LOGGER.setLevel(logging.INFO)
 LOGGER.addHandler(logging.StreamHandler(sys.stdout))
 
 
-@pytest.mark.model("mnist")
+@pytest.mark.model("sdxl")
 @pytest.mark.processor("gpu")
 @pytest.mark.gpu_test
 @pytest.mark.stabilityai_only
-def test_mnist_distributed_gpu_stabilityai_one(
-    framework_version, ecr_image, instance_type, sagemaker_regions
+def test_sdxl_v1_0_gpu_stabilityai(framework_version, ecr_image, instance_type, sagemaker_regions):
+    instance_type = instance_type or "ml.g5.4xlarge"
+    model_bucket = "stabilityai-public-packages"
+    model_prefix = "model-packages/sdxl-v1-0-dlc"
```

Comment:
This is public, but may need to be adjusted if the build environment is locked down, and might be better hosted by AWS - open to any ideas about how to handle it.